javascript/big-practice/src/scripts/models/noteModel.js

```diff
@@ -67,35 +47,24 @@ export default class NoteModel {
    * @returns {Array} listNotes
    */
   async filterNotes(type) {
-    let listNotes = [];
-    const userId = this.localStorage.getItems(STORAGE_KEYS.USER_ID);
+    const notes = await getAllNotes();
 
     // This condition filter that we can use this function for trashNotes and listNotes
     switch (type) {
-      case 'listNotes':
-        if (userId) {
-          const notes = await getDataByUserId(userId);
-          listNotes = notes.filter((note) => !note.deleteAt);
-        } else {
-          listNotes = this.notes.filter((note) => !note.deleteAt);
-        }
-
+      case 'listNotes': {
+        this.listNotes = notes.filter((note) => !note.deleteAt);
         break;
-      case 'trashNotes':
-        if (userId) {
-          const notes = await getDataByUserId(userId);
-          listNotes = notes.filter((note) => note.deleteAt);
-        } else {
-          listNotes = this.notes.filter((note) => note.deleteAt);
-        }
-
+      }
+      case 'trashNotes': {
+        this.listNotes = notes.filter((note) => note.deleteAt);
         break;
+      }
       default:
         console.log('Must enter a listNotes or trashNotes');
         break;
     }
 
-    return listNotes;
+    return this.listNotes;
   }
 
   /**
```

Comment:
Be like the way to description method deleteNoteInTrash, should updated more clearly